src/App.tsx

```diff
@@ -15,36 +17,65 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  NONE = 'none',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+}
+
 export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
+  const [vissibleGoods, setVisibleGoods] = useState<string[]>(goodsFromServer);
```

Comment:
There is a typo in the variable name `vissibleGoods`. It should be `visibleGoods` to correctly reflect its purpose and avoid confusion.